front/migrations/20250429_update_agent_scopes.ts

```diff
@@ -47,50 +48,46 @@ const migrateWorkspace = async (
 
   for (const agent of agents) {
     const previousScope = agent.scope;
-    if (
-      previousScope === "workspace" ||
-      previousScope === "published" ||
-      previousScope === "private"
-    ) {
-      const newScope = previousScope === "private" ? "hidden" : "visible";
-      logger.info(
-        {
-          workspace: workspace.sId,
-          id: agent.id,
-          agent: agent.sId,
-          version: agent.version,
-          previousScope,
-          newScope,
-        },
-        "Migrating agent scope"
-      );
+    // @ts-ignore
```

Comment:
What do we ignore? 🙏🏻

Reply:
the fact that typescript don't understand that `previousScope` can `=== 'private'` - it's not a valid value anymore, but the value is coming from db and that's the goal of the migration script to update them